src/radar_control.py

```diff
@@ -345,7 +355,7 @@ def search_for_experiment(radar_control_to_exp_handler, exphan_to_radctrl_iden,
     new_experiment_received = False
 
     try:
-        serialized_exp = socket_operations.recv_exp(
+        serialized_exp = so.recv_bytes(
```

Comment:
We can change this to `so.recv_pyobj(..., expected_type=ExperimentPrototype)` now and remove some of the if statements below.

Reply:
I didn't change to `send_pyobj` on the experiment handler side just because I'm short on time and I don't want to break anything over there.